src/metabase/lib/equality.cljc

```diff
@@ -178,7 +179,9 @@
           (when-not (next non-exprs)
             (first non-exprs))))
       ;; In all other cases, this is an ambiguous match.
-      (throw (ambiguous-match-error a-ref columns))))
+      #_(throw (ambiguous-match-error a-ref columns))
+      #?(:cljs (js/console.warn (ambiguous-match-error a-ref columns))
+         :clj  (log/warn (ambiguous-match-error a-ref columns)))))
```

Comment:
I suspected that was the reason. Isn't it in general preferable to omit `pr-str`?

Reply:
Yeah, I encouraged folks to use direct `console.log` and friends when logging debug stuff in CLJS since it's nicer to work with. But last I saw the `metabase.util.log` code does the `pr-str` for you. We could probably fix that but it hasn't happened yet.